modules/ROOT/pages/_partials/dataweave-intro.adoc

```diff
@@ -1,11 +1,3 @@
+DataWeave is the programming language designed by MuleSoft for data transformation, it is also the expression language Mule runtime engine uses to configure components and connectors.
```

Comment:
```suggestion
DataWeave is the programming language designed by MuleSoft for data transformation, it is also the expression language Mule runtime engine (Mule) uses to configure components and connectors.
```

Reply:
Might sound repetitive the sentence "_transform the data that travels through a Mule app_" in the second line but I was thinking as a new developer that does not know anything about DW or Mule...